bigquery/snippets/job/integration_test.go

```diff
@@ -93,8 +93,12 @@ func TestCopiesAndExtracts(t *testing.T) {
 		t.Errorf("copyTable(%s): %v", testDatasetID, err)
 	}
 
-	if err := copyTableWithCMEK(tc.ProjectID, testDatasetID, "copycmek"); err != nil {
-		t.Errorf("copyTableWithCMEK(%s): %v", testDatasetID, err)
+	if bqtestutil.RunCMEKTests() {
+		if err := copyTableWithCMEK(tc.ProjectID, testDatasetID, "copycmek"); err != nil {
```

Comment:
Consider a subtest with a `t.Skip` call, rather than a `t.Log` message. Same with the other tests (especially the subtest ones). Could also pass `t` to `RunCMEKTests` and have it call `Skip`.

Reply:
Switched this around.  I also grouped the copy and tests here into groups that can run in parallel.